deploy/charts/cert-manager/values.yaml

```diff
@@ -501,6 +501,20 @@ cainjector:
   # Automounting API credentials for a particular pod
   # automountServiceAccountToken: true
 
+acmesolver:
+  image:
+    repository: quay.io/jetstack/cert-manager-acmesolver
```

Comment:
suggestion: Would it be possible for us to document the new fields in https://github.com/cert-manager/cert-manager/blob/master/deploy/charts/cert-manager/README.template.md ?

Reply:
Sure!

Reply:
Thanks! I'm just finishing for the day, so I'll take a look tomorrow. Thanks for your efforts on this :tada: